packages/docs/versioned_docs/version-2.0.0-alpha-3/protocol/07-nullifiers.md

```diff
@@ -1,13 +1,13 @@
 # Nullifiers
 
-* Nullifiers are used to prevent things from happening more than once
-* UniRep uses two different nullifiers
+Nullifiers are used to prevent things from happening more than once. UniRep uses two different nullifiers:
 
 ## Epoch key nullifiers
 
-* Epoch key nullifiers are used to prevent users from using the same epoch key twice and prevent users from double [user state transition](05-user-state-transition.md).
-* If an epoch key nullifier is seen before, then drop the second user state transition proof.
-* Nullifier of an epoch key is computed by
+* Ensure a user's epoch keys are unique from any previous epoch
```

Comment:
another re-wording: is my interpretation accurate?

Reply:
No, epoch key nullifiers are used to prevent doing a user state transition multiple times.

Reply:
<img width="1144" alt="Screen Shot 2022-12-20 at 1 33 40 PM" src="https://user-images.githubusercontent.com/79813968/208761215-a02a2c6b-0c01-4f99-9e18-c3f4918ae2bd.png">
<img width="1144" alt="Screen Shot 2022-12-20 at 1 34 00 PM" src="https://user-images.githubusercontent.com/79813968/208761222-3ff6b60c-40ef-41ce-88fa-a92825ec7285.png">

Here is the existing wording.. I separated this line into 2 points - my new lines 7 and 8.
Looks like new line 8 is accurate, but as 7 is not, can you clarify the meaning of the first part of the original.. "used to prevent users from using the same epoch key twice"

Reply:
Should this read:
"Epoch key nullifiers prevent users from using an epoch key to perform the same state transition twice." 
?